zerver/openapi/zulip.yaml

```diff
@@ -14997,6 +15031,11 @@ paths:
                               type: string
                               description: |
                                 The name of the external account provider
+                            rendered_name:
+                              type: string
+                              description: |
+                                The markdown renderedname of the external account
+                                provider
```

Comment:
You would need to add "Changes" entry here. You can search for "Changes" in the file to see how it is done.